ppx/browser/dune

```diff
@@ -0,0 +1,48 @@
+(library
+ (public_name melange-json.browser)
+ (name ppx_deriving_json_browser)
+ (modules
+  :standard
+  \
+  ppx_deriving_json_runtime
+  ppx_deriving_json_browser_test)
+ (libraries ppxlib containers)
+ (ppx_runtime_libraries melange-json.browser_runtime)
+ (preprocess
+  (pps ppxlib.metaquot))
+ (kind ppx_deriver))
+
+(library
+ (public_name melange-json.browser_runtime)
+ (name ppx_deriving_json_browser_runtime)
+ (modules ppx_deriving_json_runtime)
+ (wrapped false)
```

Comment:
is this a wrapped library to reuse the `ppx_deriving_json_runtime` module?

Reply:
yes, so the same code could be compiled for native and js